di-ipv-orchestrator-stub/deploy-core-dev/template.yaml

```diff
@@ -76,6 +76,14 @@ Parameters:
     Description: "Environment Variables for ECS - Retrieved from SSM Parameter Store"
     Type: AWS::SSM::Parameter::Value<String>
     Default: "/stubs/build/orch/env/JBP_CONFIG_OPEN_JDK_JRE" #pragma: allowlist secret
+  OrchestratorEnableBasicAuth:
+    Description: "Environment Variables for ECS"
+    Type: String
+    Default: "true" #pragma: allowlist secret
+  OrchestratorBasicAuthCredentials:
+    Description: "Environment Variables for ECS - Retrieved from SSM Parameter Store"
+    Type: String
+    Default: "{ \"username\": \"dev\", \"password\": \"dev\" }" #pragma: allowlist secret
```

Comment:
If this is too insecure we can change it match the other templates. This would mean that developers would need to add the relevant SSM parameter before they could re-deploy their orch stubs.